xsd/netex-nl.xsd

```diff
@@ -98,55 +98,55 @@
-				<xsd:attribute name="ref" type="BisonTypeOfProfileFrame" use="required" fixed="BISON:TypeOfFrame:NL_TT_INFRA" form="unqualified"/>
+				<xsd:attribute name="ref" type="BisonTypeOfInfrastructureFrame" use="required" form="unqualified"/>
 			</xsd:restriction>
 		</xsd:complexContent>
 	</xsd:complexType>
 	<xsd:complexType name="TypeOfFrameRefStructure-service">
 		<xsd:complexContent>
 			<xsd:restriction base="TypeOfFrameRefStructure">
 				<xsd:attribute name="version" type="VersionIdType" use="required" form="unqualified"/>
-				<xsd:attribute name="ref" type="BisonTypeOfProfileFrame" use="required" fixed="BISON:TypeOfFrame:NL_TT_SERVICE" form="unqualified"/>
+				<xsd:attribute name="ref" type="BisonTypeOfServiceFrame" use="required" form="unqualified"/>
 			</xsd:restriction>
 		</xsd:complexContent>
 	</xsd:complexType>
 	<xsd:complexType name="TypeOfFrameRefStructure-timetable">
 		<xsd:complexContent>
 			<xsd:restriction base="TypeOfFrameRefStructure">
 				<xsd:attribute name="version" type="VersionIdType" use="required" form="unqualified"/>
-				<xsd:attribute name="ref" type="BisonTypeOfProfileFrame" use="required" fixed="BISON:TypeOfFrame:NL_TT_TIMETABLE" form="unqualified"/>
+				<xsd:attribute name="ref" type="BisonTypeOfTimetableFrame" use="required" form="unqualified"/>
 			</xsd:restriction>
 		</xsd:complexContent>
 	</xsd:complexType>
 	<xsd:complexType name="TypeOfFrameRefStructure-calendar">
 		<xsd:complexContent>
 			<xsd:restriction base="TypeOfFrameRefStructure">
 				<xsd:attribute name="version" type="VersionIdType" use="required" form="unqualified"/>
-				<xsd:attribute name="ref" type="BisonTypeOfProfileFrame" use="required" fixed="BISON:TypeOfFrame:NL_TT_CALENDAR" form="unqualified"/>
+				<xsd:attribute name="ref" type="BisonTypeOfServiceCalendarFrame" use="required" form="unqualified"/>
 			</xsd:restriction>
 		</xsd:complexContent>
 	</xsd:complexType>
 	<xsd:complexType name="TypeOfFrameRefStructure-vehicle">
 		<xsd:complexContent>
 			<xsd:restriction base="TypeOfFrameRefStructure">
 				<xsd:attribute name="version" type="VersionIdType" use="required" form="unqualified"/>
-				<xsd:attribute name="ref" type="BisonTypeOfProfileFrame" use="required" fixed="BISON:TypeOfFrame:NL_TT_VEHICLE" form="unqualified"/>
+				<xsd:attribute name="ref" type="BisonTypeOfVehicleScheduleFrame" use="required" form="unqualified"/>
 			</xsd:restriction>
 		</xsd:complexContent>
 	</xsd:complexType>
 	<xsd:complexType name="TypeOfFrameRefStructure-site">
 		<xsd:complexContent>
 			<xsd:restriction base="TypeOfFrameRefStructure">
 				<xsd:attribute name="version" type="VersionIdType" use="required" form="unqualified"/>
-				<xsd:attribute name="ref" type="BisonTypeOfProfileFrame" use="required" fixed="BISON:TypeOfFrame:NL_TT_SITE" form="unqualified"/>
+				<xsd:attribute name="ref" type="BisonTypeOfSiteFrame" use="required" form="unqualified"/>
```

Comment:
Ik snap niet zo goed hoe je dit hebt gedaan. Wat is de reden om die waardes niet meer 'fixed' it maken?

Reply:
Voor ResourceFrame en InfrastructureFrame zijn er nu twee mogelijke waardes. Je kan in de attribuut-definitie dan dus niet langer een fixed waarde opgeven. Maar om niet meteen alle waardes uit de TypeOfFrame-enumeratie (die we hadden) toe te staan, heb ik de opbouw van die TypeOfFrames wat anders gemaakt.

Feitelijk heb je nu per frame een eigen enumeratie, met daarin alleen de waardes die relevant zijn. Voor ResourceFrame en InfrastructureFrame zijn dat er twee. Voor bijvoorbeeld het SiteFrame is het er nog steeds maar ééntje, en is het op die manier nog steeds 'fixed'.